tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py

```diff
@@ -179,43 +179,83 @@ def create_run_config(
 
     run_azuremlcompute = RunConfiguration()
     run_azuremlcompute.target = cpu_cluster
-    run_azuremlcompute.environment.docker.enabled = True
-    run_azuremlcompute.environment.docker.base_image = docker_proc_type
-
-    # Use conda_dependencies.yml to create a conda environment in
-    # the Docker image for execution
-    # False means the user will provide a conda file for setup
-    # True means the user will manually configure the environment
-    run_azuremlcompute.environment.python.user_managed_dependencies = False
 
-    # install local version of recommenders on AzureML compute using .whl file
-    whl_url = run_azuremlcompute.environment.add_private_pip_wheel(
-        workspace=workspace,
-        file_path=reco_wheel_path,
-        exist_ok=True,
-    )
-    conda_dep = CondaDependencies()
-    conda_dep.add_conda_package(conda_pkg_python)
-    conda_dep.add_pip_package(whl_url)
-    conda_dep.add_pip_package(
-        "pymanopt@https://github.com/pymanopt/pymanopt/archive/fb36a272cdeecb21992cfd9271eb82baafeb316d.zip"
-    )
-
-    # install extra dependencies
-    if add_gpu_dependencies and add_spark_dependencies:
-        conda_dep.add_channel("conda-forge")
-        conda_dep.add_conda_package(conda_pkg_jdk)
-        conda_dep.add_pip_package("recommenders[dev,examples,spark,gpu]")
-    elif add_gpu_dependencies:
-        conda_dep.add_pip_package("recommenders[dev,examples,gpu]")
-    elif add_spark_dependencies:
-        conda_dep.add_channel("conda-forge")
-        conda_dep.add_conda_package(conda_pkg_jdk)
-        conda_dep.add_pip_package("recommenders[dev,examples,spark]")
-    else:
-        conda_dep.add_pip_package("recommenders[dev,examples]")
+    # Enable DockerSection https://learn.microsoft.com/en-us/python/api/azureml-core/azureml.core.environment.dockersection
+    run_azuremlcompute.environment.docker.enabled = True
+    # run_azuremlcompute.environment.docker.base_image = docker_proc_type
+    
+    dockerfile = """
```

Comment:
I think it is more reliable to use `CondaDependencies` to set up the Conda env than dockerfile

Reply:
@miguelgfierro I see why you tried to use dockerfile here.  I think it should be possible to use `CondaDependencies` to install recommenders locally.  I'll do a research.